docs/en/observability/inspect-uptime-duration-anomalies.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[inspect-uptime-duration-anomalies]]
+= Inspect uptime duration anomalies
+
+When the {anomaly-detect} features of {ml} are enabled,
```

Comment:
Might be worth pointing to docs about enabling this feature, or explaining how to tell if it's enabled. I find the UI (7.9.2, at least) to be less than helpful. If you don't have ML enabled, but click the link, you see flyout with a vague description, and you have to follow a link and notice the message at the top of the page. I can figure out from the message that I need to edit my deployment, but it's easy to overlook.